bootstrap-demo/kubernetes-dns/kubernetes/akka-cluster.yml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: "apps/v1beta2"
+kind: Deployment
+metadata:
+  name: "bootstrap-demo-kubernetes-dns-v1-3-3-7"
+  labels:
+    appName: "bootstrap-demo-kubernetes-dns"
+    appNameVersion: "bootstrap-demo-kubernetes-dns-v1-3-3-7"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      appNameVersion: "bootstrap-demo-kubernetes-dns-v1-3-3-7"
+  template:
+    metadata:
+      labels:
+        appName: "bootstrap-demo-kubernetes-dns"
+        appNameVersion: "bootstrap-demo-kubernetes-dns-v1-3-3-7"
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: "bootstrap-demo-kubernetes-dns"
+        image: "bootstrap-demo-kubernetes-dns:1.3.3.7"
+        imagePullPolicy: IfNotPresent
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: 8080
+        livenessProbe:
+          httpGet:
+            path: /alive
+            port: 8080
+        env:
+        ports:
+        - containerPort: 8558
+          name: management
+        - containerPort: 2552
+          name: remoting
+        volumeMounts: []
+      volumes: []
+---
+#headless
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    appName: "bootstrap-demo-kubernetes-dns"
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  name: "bootstrap-demo-kubernetes-dns"
+spec:
+  ports:
+  - name: management
+    port: 8558
+    protocol: TCP
+    targetPort: 8558
+  - name: remoting
+    port: 2552
+    protocol: TCP
+    targetPort: 2552
+  selector:
+    appName: "bootstrap-demo-kubernetes-dns"
+  clusterIP: None
+  publishNotReadyAddresses: true
+#headless
+---
+#public
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    appName: "bootstrap-demo-kubernetes-dns"
+  name: "bootstrap-demo-kubernetes-dns-public"
```

Comment:
maybe remove the `-public` suffix here and add a `-headless` or `-akka-internal` or something similar to the headless service?

Reply:
Yep good idea